src/packages/src/xroad/ubuntu/generic/control

```diff
@@ -129,12 +127,6 @@ Depends: ${misc:Depends}, xroad-proxy (=${binary:Version}), xroad-opmonitor (=${
 Description: X-Road AddOn: operations monitoring service
  AddOn for operations monitoring.
 
-Package: xroad-center-clusterhelper
```

Comment:
This does not exist anymore? Not directly related to this ticket, I guess?

Reply:
That sets up a cluster using BDR 1.0, which is not supported any more.